CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ Please add to relevant subsections here on every PR where this is applicable.
 * Preliminary support for forking a run using `wandb.init(fork_from=...)` by @dannygoldstein in https://github.com/wandb/wandb/pull/7078
 * `run.save()` accepts `pathlib.Path` values; by @timoffex in https://github.com/wandb/wandb/pull/7146
 * Reduced default status print frequency of launch agent. Added verbosity controls to allow for increased status print frequency and printing debug information to stdout by @TimH98 in https://github.com/wandb/wandb/pull/7126
+* Added support for overriding kaniko builder settings in the agent config by @TimH98 in https://github.com/wandb/wandb/pull/7191
```

Comment:
Thanks for updating the change log, it is very useful to inform our users. however you are adding it under a section that is already released :)
you should add your changes under the `Unreleased` section that would be updated with the correct version/date when we do the release.

Reply:
Gah thanks good catch, moved to unreleased